lib/utils/fields.js

```diff
@@ -187,8 +187,12 @@ export function slotNamesForTitlesMap(fields) {
   const fieldSymbolsAtIndex = fieldSymbolsAtIndexMap(fields);
   let tempObject = {};
   for (let index in fieldSymbolsAtIndex) {
-    tempObject[fieldSymbolsAtIndex[index].slotTitle] =
-      fieldSymbolsAtIndex[index].slotName;
+    const slotName = fieldSymbolsAtIndex[index].slotName;
+    let slotTitle = fieldSymbolsAtIndex[index].slotTitle;
+    // If slotTitle not specified; use slotName as slotTitle
+    slotTitle = slotTitle === undefined ? slotName : slotTitle;
+
+    tempObject[slotTitle] = slotName;
```

Comment:
Is this what we want? A test was failing previously because the `fields` global var in `fields.test.js` was missing `slotTitle` values. There are several lines of code downstream of this function that expect `slotTitle` values. I just re-used `slotName` as title as well to fix things.

Reply:
Remove references to slotTitle. We only want slotName